tmk_core/protocol/lufa/lufa.c

```diff
@@ -1023,6 +1027,9 @@ int main(void) {
 #endif
 
     setup_mcu();
+#ifdef EEPROM_DRIVER
+    eeprom_driver_init();
+#endif
```

Comment:
Another part of EEPROM code which does not seem to belong here.

Reply:
See line comment on line 95 above.